Domains/Smartmetering/smartmeteringwebservices/GetConfigurationObject.md

```diff
@@ -0,0 +1,25 @@
+## GetConfigurationObject request
+
+### Description
+GetConfigurationObject is a request to retrieve a ConfigurationObject from on a device. The attributes with OBIS code 0-1:94.31.3.255 is used to access the GPRS_operation_mode setting and following flags:
```

Comment:
The end of the first sentence looks like the "on" should not be there in "from on a device".